### ISSUE TITLE
Immutable objects detected as mutable when immutable.

### ISSUE THREAD

**Issue by envygeeks** [bbatsov/rubocop#2630]
`master` warns about freezing mutable `Object`'s but does not detect the `frozen-string-literal: true` comment leading to an invalid warning:

````
lib/forwardable/extended/version.rb:7:15: C: Freeze mutable object assigned to constants
    VERSION = "2.0.0"
              ^^^^^^^

2 files inspected, 4 offenses detected
jordon ~/development (master *) % pry
[1] pry(main)> Forwardable::Extended::VERSION.frozen?
=> true
```

**Comment by envygeeks** [comment]
Wow, I was confused for a second.  Either way the error is still valid.

**Comment by segiddins** [comment]
Now that the frozen string literal stuff is in master, I can try and tackle this.

**Comment by envygeeks** [comment]
Awesome, thanks!

**Comment by BM5k** [comment]
FYI: this is still a problem unless you set `TargetRubyVersion`
